### ISSUE TITLE
Improve the loss function

### ISSUE THREAD

**Issue by wetherc** [riffatahmad/Data-Science-Project#6]
Since we have a severe class imbalance, our model is probably a little lopsided in its predictions currently. One option for correcting this is to upsample the minority class (duplicate "true fraud" records in our training data). But we can also try using different loss functions. Let's see if Focal Loss gives us any better results: https://github.com/facebookresearch/Detectron

It looks like there are a couple of implementations for tensorflow floating around out there:

- https://github.com/zhezh/focalloss/blob/master/focalloss.py
- https://github.com/ailias/Focal-Loss-implement-on-Tensorflow/blob/master/focal_loss.py
